test/features/copy-in-copy-out.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Copy-in and Copy-out Support
+    In order to facilitate easier file transfer between the host and guest,
+    transient supports copy-in and copy-out support. Given a path on the host and an
+    absolute path on the VM, transient can:
+    - copy the host file or directory to the guest directory before starting the VM
+    - copy the guest file or directory to the host directory after stopping the VM
+
+  @skip-in-ci
+  Scenario: Copy in directory before starting VM
+     Given a transient vm
+       And a name "test-vm"
+       And a disk image "generic/alpine38:v3.0.2"
+       And a host directory: "artifacts/host/copy-in-test/"
+       And a guest directory: "/home/vagrant/"
+       And the host directory is copied to the guest directory before starting
+       And the vm is prepare-only
+      When the transient command is run
+      Then the return code is 0
+       And the host directory exists in the guest directory
+
+  @skip-in-ci
+  Scenario: Copy out directory after stopping VM
+     Given a transient vm
+       And a name "test-vm"
+       And a disk image "generic/alpine38:v3.0.2"
+       And a host directory: "artifacts/host/copy-out-test/"
+       And a guest directory: "/home/vagrant/"
+       And the guest directory is copied to the host directory after stopping
+       And a ssh command "echo 'dummy command'"
+      When the vm runs to completion
+      Then the return code is 0
+       And the guest directory exists in the host directory
```

Comment:
I think I'd like to test this a bit differently. Instead of running virt-copy-in and virt-copy-out in the tests, we should probably just using `-copy-in-before` to add a file like `my-test-file` to the guest home directory, the run a guest with ssh command `ls` and assert that `my-test-file` is in stdout.

Similarly, to test `-copy-out-after` we can run an ssh command that does a `touch my-test-file` and then copy that out and assert that the file exists on the host after.